apidoc/Facebook.yml

```diff
@@ -1195,12 +1249,29 @@ properties:
     since: "5.4.0"
     platforms: [iphone, ipad]
 
+  - name: loginBehavior
+    summary: |
+        Defines the behavior that the module will attempt to use for authorize().
+    description: |
+        Be sure to set this before calling authorize if this is needed. It is
+        recommended to follow Facebook's [guidelines](https://developers.facebook.com/blog/post/2015/10/29/Facebook-Login-iOS9)
+        for ideal login behavior
+    type: Number
+    constants: [LOGIN_BEHAVIOR_*]
+    default:
+        Defaults to <Titanium.Modules.Facebook.LOGIN_BEHAVIOR_NATIVE_WITH_FALLBACK>
+        on Android and <Titanium.Modules.Facebook.LOGIN_BEHAVIOR_NATIVE> on iOS
```

Comment:
The default was being set to browser to fix an issue where authorize does nothing if the native FB app is installed on the device.  I removed that so that the default behavior will remain the same as it is now (native with fallback).  The issue can be corrected in javascript by setting the loginBehavior to native.  I don't know the correct protocol for submitting the authorize issue as a bug.